src/platform/user/authentication/components/SignInModal.jsx

```diff
@@ -44,6 +36,13 @@ class SignInModal extends React.Component {
     }
   }
 
+  loginHandler = loginType => () => {
+    const { useSSOe } = this.props;
+    const evt = `login-attempted${useSSOe ? '-ssoe' : ''}-${loginType}`;
```

Comment:
sidenote: we'll need to add this to Google tag manager for it to track the `-ssoe` add-on. Also, this depends on rollout strategy.

Also, we'll need to add this same logic in `src/platform/user/authentication/components/SignInModal.jsx`

Reply:
it should also get tacked onto the `recordEvent` call in `src/applications/login/containers/SignInApp.jsx`, just without the conditional

Reply:
yeah I wasn't sure on this one, do we want a different event for ssoe vs idme login?

Reply:
I saw you took it out. given that we're gonna ramp up users pretty fast, probably ok to omit the "-ssoe" tag for now